paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/ReadWriteTableITCase.java

```diff
@@ -1331,6 +1334,22 @@ public void testStreamingInsertOverwrite() {
                 .hasMessage("Paimon doesn't support streaming INSERT OVERWRITE.");
     }
 
+    @Test
+    public void testPhysicalColumnComments() {
```

Comment:
add a case for `testComputedColumnComments` too? We would see no comments return.